.github/workflows/trigger_jenkins.yaml

```diff
@@ -0,0 +1,49 @@
+name: Trigger next gating
+
+on:
+  push:
+    branches:
+      - next**
+
+jobs:
+  trigger-jenkins:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Determine Jenkins Job Name
+        run: |
+          if [[ "${{ github.ref_name }}" == "next" ]]; then
+            FOLDER_NAME="scylla-master"
+          elif [[ "${{ github.ref_name }}" == "next-enterprise" ]]; then
```

Comment:
I think we don't have here `next-enterprise` branch..
Do we need it here? We can have more simple workflow.. Or you want to keep it the same as for all other repos?